subt/ros/robot/launch/r2.launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<launch>
+  <node name="$(arg robot_name)_proxy" pkg="proxy" type="ros_proxy" args="$(arg robot_name)" output="screen">
+    <remap from="$(arg robot_name)/front/depth" to="$(arg robot_name)/rs_front/depth/image"/>
```

Comment:
Why do we have this remap here? We are using c2o to send the data, right? Is it because of the waitForMessage? If so, I'd vote for removing that wait instead.